src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
+  .menu-bars {
+    position: fixed;
+    left: calc(50% - 134px);
+    bottom: 8px;
+  }
+  .menu-bars--isAdmin {
+    left: calc(50% - 168px);
+  }
+  .menu-bars--user {
+    @media (prefers-color-scheme: light) {
+      .menu-bars__switch {
+        background-color: #FFFFFF;
```

Comment:
Please define global variables for the color codes that are used within this css file

Reply:
Done ✅